libcontainer/integration/exec_test.go

```diff
@@ -705,6 +705,150 @@ func testRunWithKernelMemory(t *testing.T, systemd bool) {
 	}
 }
 
+func TestCgroupResourcesUnifiedErrorOnV1(t *testing.T) {
+	testCgroupResourcesUnifiedErrorOnV1(t, false)
+}
+
+func TestCgroupResourcesUnifiedErrorOnV1Systemd(t *testing.T) {
+	if !systemd.IsRunningSystemd() {
+		t.Skip("Systemd is unsupported")
+	}
+	testCgroupResourcesUnifiedErrorOnV1(t, true)
+}
+
+func testCgroupResourcesUnifiedErrorOnV1(t *testing.T, systemd bool) {
+	if testing.Short() {
+		return
+	}
+	if cgroups.IsCgroup2UnifiedMode() {
+		t.Skip("requires cgroup v1")
+	}
+	rootfs, err := newRootfs()
+	ok(t, err)
+	defer remove(rootfs)
+
+	config := newTemplateConfig(rootfs)
+	if systemd {
+		config.Cgroups.Parent = "system.slice"
+	}
+	config.Cgroups.Resources.Unified = map[string]string{
+		"memory.min": "10240",
+	}
+	_, _, err = runContainer(config, "", "true")
+	if !strings.Contains(err.Error(), cgroups.ErrV1NoUnified.Error()) {
+		t.Fatalf("expected error to contain %v, got %v", cgroups.ErrV1NoUnified, err)
+	}
+}
+
+func TestCgroupResourcesUnified(t *testing.T) {
+	testCgroupResourcesUnified(t, false)
+}
+
+func TestCgroupResourcesUnifiedSystemd(t *testing.T) {
+	if !systemd.IsRunningSystemd() {
+		t.Skip("Systemd is unsupported")
+	}
+	testCgroupResourcesUnified(t, true)
+}
+
+func testCgroupResourcesUnified(t *testing.T, systemd bool) {
```

Comment:
We should also have integration tests

Reply:
Sure, will do

Reply:
Added a few basic tests. Let me know if you had something else in mind.